doc/source/reference/module_structure.rst

```diff
@@ -5,7 +5,7 @@ NumPy's module structure
 ************************
 
 NumPy has a large number of submodules. Most regular usage of NumPy requires
-only the main namespace and a smaller set of submodules. The rest either either
+only the main namespace and a smaller set of submodules. The rest either has
```

Comment:
has -> have

Reply:
I'm not native english, I thought _The reset_ is singular so ist must be _has_.